src/app/projects/create-new-project/create-new-project.component.html

```diff
@@ -18,13 +18,19 @@ <h5 class=" gray-text"><b>Create a new project</b></h5>
 
         </div>
     </div>
-    <form [formGroup]="createProjectForm" class="mb-5">
+    <form [formGroup]="createForm" class="mb-5">
         <div class="explainer-mini  mb-2 mt-3">Project name</div>
         <input class="w-100 p-4 form-control " formControlName="name" id="" name="" type="text">
 
+        <div class="explainer-mini  mb-2 mt-3">Project description</div>
+        <textarea class="form-control p-4" cols="30" id="project-description" name=""
+                  formControlName="description"
+                  rows="10" style="border-radius: 5px!important;">
+        </textarea>
+
         <div class="explainer-mini  mb-2 mt-3">Project location</div>
-        <app-location-map [movableMarker]="true" 
-            [coordsSet]="projectCoords" (getMapCoords)="getMapCoords($event)">
+        <app-location-map [editable]="true"
+                          [(lat)]="mapLat" [(lng)]="mapLong">
```

Comment:
We tend to have a simple API for using our component. On your side, it's unclear what this component does with these events and `coordsSet`. 

What we need in this case is being able to attach latitude and longitude and being able to edit them.
With [Angular two-way binding](https://angular.io/guide/two-way-binding), we can directly map the changes. Check it out.